### DISCUSSION TITLE
Allow Dashboard List to sort by starred

### DISCUSSION THREAD

**Discussion by jp-moller** [grafana/grafana#59258] [Dashboards, Panels & Visualizations]
Hi team,

We currently have multiple different folders for each team within the organisation. We have created a "Home" Dashboard within our folder that nicely sorts all of our dashboards into relevant dashboard lists. 

The only issue is that everyone in the team uses certain dashboards more frequently than others. To avoid creating a "Home" Dashboard for each team member, we want to create one unified dashboard where we can sort the "Dashboard List" panels by starred dashboards. This way, we can all share one home, but also see the more relevant dashboards and not having to scroll to find the ones each team member uses more frequently.

Could I please ask for a sorting filter option on the "Dashboard List" panel to allow sorting by starred?
![image](https://user-images.githubusercontent.com/29963763/203658492-2323ea87-8fef-4ce9-a0d8-e8202706978e.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
